components/org.wso2.carbon.identity.branding.preference.management.core/src/test/java/org/wso2/carbon/identity/branding/preference/management/core/util/MockUIBrandingPreferenceResolver.java

```diff
@@ -37,6 +37,10 @@ public BrandingPreference resolveBranding(String type, String name, String local
     }
 
     @Override
+    public void clearBrandingResolverCacheHierarchy(String currentTenantDomain) throws BrandingPreferenceMgtException {
+        setBranding(null);
+    }
```

Comment:
missing new line after the method signature

Reply:
Addressed with https://github.com/wso2-extensions/identity-branding-preference-management/pull/38